src/image_surface.rs

```diff
@@ -15,25 +16,29 @@ use ::enums::{
 };
 
 use BorrowError;
-use surface::{Surface, SurfaceExt};
+use surface::Surface;
 use Status;
 use std::fmt;
 
 #[derive(Debug)]
 pub struct ImageSurface(Surface);
 
-impl ImageSurface {
-    pub fn from(surface: Surface) -> Result<ImageSurface, Surface> {
+impl TryFrom<Surface> for ImageSurface {
```

Comment:
Should make sure in the release notes that we communicate that we only support 1.34+ now.